quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/widget/WidgetController.java

```diff
@@ -22,23 +25,25 @@
 @RequestMapping("/api/widgets")
 public class WidgetController {
 
-    private WidgetConfigService widgetService;
-    private WidgetConfigMapper  widgetConfigMapper;
-    private SourceDataMapper    sourceDataMapper;
+    private WidgetConfigRepository widgetConfigRepository;
+    private WidgetConfigMapper     widgetConfigMapper;
+    private SourceDataMapper       sourceDataMapper;
+    private SourceDataRepository   sourceDataRepository;
 
     @Autowired
-    public WidgetController(WidgetConfigService widgetService,
+    public WidgetController(WidgetConfigRepository widgetConfigRepository,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)